script/devsds/lib/opensds.sh

```diff
@@ -25,7 +25,6 @@ osds:opensds:configuration(){
 cat >> $OPENSDS_CONFIG_DIR/opensds.conf << OPENSDS_GLOBAL_CONFIG_DOC
 [osdsapiserver]
 api_endpoint = 0.0.0.0:50040
-log_file = /var/log/opensds/osdsapiserver.log
```

Comment:
If you want to remove this line, please change other related files.

Reply:
done